pyctcdecode/tests/test_language_model.py

```diff
@@ -68,6 +68,27 @@ def test_trie(self):
         has_token = char_trie.has_node("U.S") > 0
         self.assertTrue(has_token)
 
+    def test_load_from_hub(self):
```

Comment:
this test makes sure the hashing logic is correct and does **not** require internet connection.